brigade/templates/base.html

```diff
@@ -48,17 +46,17 @@
 
     </div>
 
+    <script>
```

Comment:
I would suggest we wrap the Google Analytics code in a {% block analytics %} so we have the option to override it in a particular template if we ever needed to.

Reply:
Will do, thanks.